src/docs/markdown/caddyfile/matchers.md

```diff
@@ -239,7 +306,40 @@ not {
 
 The results of the enclosed matchers will be negated.
 
+#### Examples:
+
+Match requests with paths that do NOT begin with `/css/` OR `/js/`.
+
+```caddy-d
+not path /css/* /js/*
+```
+
+Match requests WITH NEITHER:
+- an `/api/` path prefix, NOR
+- the `POST` request method
+
+i.e. can have none of these to match
```

Comment:
```suggestion
i.e. cannot match either of these:
```

Reply:
I don't think I agree - the entire thing is the matcher, not those individually. Agree on the `:` though.

Reply:
Well they are also matchers, it is matchers within matchers. I don't think people will care too much, do you think they won't know what we mean when we say "both"?

Reply:
The entire point of the example is to explain the behaviour of multiple matchers with `not`s in the context of a named matcher. IMO we should be consistent here when using the word `match` to always refer to the same thing (the named matcher) instead of either referring to the individual matchers or the named matcher.

Reply:
Uh, well how about we call them "inner" matchers then? I know it's confusing, but these are advanced cases, after all. At some point, some thinking is required. :)

I just think "can have none of these to match" is both awkward English and more confusing than "cannot match either of these" -- I assume most people will be able to know what "these" refers to?

Reply:
I was the one who suggested "can have none of these to match". It's not technically incorrect, but you're right, it sounds awkward because that kind of phrasing is not common at all in conversation.

Perhaps "must have none of these to match". The intent was to follow from the wording from earlier: "can have zero or one of these to match", which could also be changed to "must".

Reply:
I was just about to suggest that @whitestrake 👍